tests/mock/DataObjectAnnotatorTest_Team.php

```diff
@@ -1,42 +1,48 @@
-<?php
-
-/**
- *
- */
-
-/* comment */
-
-// Another comment
-class DataObjectAnnotatorTest_Team extends DataObject implements TestOnly
-{
-
-    private static $db = array(
-        'Title'      => 'Varchar',
-        'VisitCount' => 'Int',
-        'Price'      => 'Currency'
-    );
-
-    private static $has_one = array(
-        "Captain"            => 'DataObjectAnnotatorTest_Player',
-        'HasOneRelationship' => 'DataObjectAnnotatorTest_Player',
-    );
-
-    private static $has_many = array(
-        'SubTeams' => 'DataObjectAnnotatorTest_SubTeam',
-        'Comments' => 'DataObjectAnnotatorTest_TeamComment'
-    );
-
-    private static $many_many = array(
-        'Players'           => 'DataObjectAnnotatorTest_Player.Players',
-        'Reserves'          => 'DataObjectAnnotatorTest_Player.Reserves',
-        'SecondarySubTeams' => 'DataObjectAnnotatorTest_SubTeam',
-    );
-
-    public function SecondarySubTeams()
-    {
-
-    }
-
-}
-
-Config::inst()->update('DataObjectAnnotatorTest_Team', 'extensions', array('DataObjectAnnotatorTest_Team_Extension'));
+<?php
+
+namespace Axyr\IDEAnnotator\Tests;
+
+use SilverStripe\Core\Config\Config;
+use SilverStripe\ORM\DataObject;
+use SilverStripe\Dev\TestOnly;
+
+/**
+ *
+ */
+
+/* comment */
+
+// Another comment
+class Team extends DataObject implements TestOnly
+{
+
+    private static $db = array(
+        'Title'      => 'Varchar',
+        'VisitCount' => 'Int',
+        'Price'      => 'Currency'
+    );
+
+    private static $has_one = array(
+        "Captain"            => 'Axyr\IDEAnnotator\Tests\Player',
+        'HasOneRelationship' => 'Axyr\IDEAnnotator\Tests\Player',
+    );
+
+    private static $has_many = array(
+        'SubTeams' => 'Axyr\IDEAnnotator\Tests\SubTeam',
+        'Comments' => 'Axyr\IDEAnnotator\Tests\TeamComment'
+    );
+
+    private static $many_many = array(
+        'Players'           => 'Axyr\IDEAnnotator\Tests\Player.Players',
+        'Reserves'          => 'Axyr\IDEAnnotator\Tests\Player.Reserves',
+        'SecondarySubTeams' => 'Axyr\IDEAnnotator\Tests\SubTeam',
+    );
+
+    public function SecondarySubTeams()
+    {
+
+    }
+
+}
+
+Config::modify()->set('Axyr\IDEAnnotator\Tests\Team', 'extensions', array('Axyr\IDEAnnotator\Tests\Team_Extension'));

```

Comment:
This should be refactored and put into `$extensions` somewhere on a class, or `$required_extensions` on a SapphireTest instance

Reply:
Yep, all still old stuff :)